src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -917,12 +928,10 @@ void ScenesServer::HandleStoreScene(HandlerContext & ctx, const Commands::StoreS
     CHIP_ERROR err = StoreSceneParse(ctx.mCommandHandler.GetAccessingFabricIndex(), ctx.mRequestPath.mEndpointId, req.groupID,
                                      req.sceneID, mGroupProvider);
 
-    if (CHIP_NO_ERROR == err)
-    {
-        ReturnOnFailure(UpdateLastConfiguredBy(ctx, response));
-    }
+    VerifyOrReturn(CHIP_NO_ERROR == AddResponseOnError(ctx, response, err));
```

Comment:
Given that you use `ReturnOnFailure` below, should this maybe also be `ReturnOnFailure(AddResponseOnError(ctx, response, err));` ?